### ISSUE TITLE
Set PLA on by default

### ISSUE THREAD

**Issue by Raph692** [OasisLMF/OasisUI#317]
Hello,

Our models require PLA factors to be applied by default to capture additional per-asset costs of wind farms. Users currently apply PLA factors by ticking the PLA box in the Oasis UI:
![image](https://github.com/OasisLMF/OasisUI/assets/72822393/5f472fbb-ebba-428a-aded-6f53c2fdcc38)

We were wondering if there's any way for model vendors to enable the PLA box to be ticked by default (to avoid the risk of users forgetting to expand the "Advanced" button to tick the PLA box)?

Many thanks!

**Comment by Raph692** [comment]
Yes, I agree. It is best not to have PLA always on, and would be preferable to have an on/off flag in the model settings that will set it to that state in the UI by default. 

I can't see the PLA flag in the current model settings schema on github - I assume you're suggesting to add this in the future?

**Comment by benhayes21** [comment]
yes
